### PR TITLE
Refactor: Change any type usage and refactor the code

### PR DESCRIPTION
**Pull request by v-tsun**

### CODE REVIEW

**Comment by v-tsun** [comment]
@rakku-ms

**Comment by v-tsun** [comment]
@bganapa

**Comment by v-tsun** [comment]
@sijuman

File: src/tree/StorageAccountTreeItem.ts [hunk 1]
```diff
@@ -126,7 +128,8 @@ export class StorageAccountTreeItem extends AzureParentTreeItem<IStorageRoot> {
         const result = await ext.ui.showWarningMessage(message, { modal: true }, DialogResponses.deleteResponse, DialogResponses.cancel);
         if (result === DialogResponses.deleteResponse) {
             const deletingStorageAccount: string = localize('deletingStorageAccount', 'Deleting storage account "{0}"...', this.label);
-            let storageManagementClient = createAzureClient(this.root, StorageManagementClient);
+            let clientResult = await createStorageClientResult(this.root, false);
+            let storageManagementClient = clientResult.clinet;
```
Comment (bganapa) [inline src/tree/StorageAccountTreeItem.ts:132]: typo client ,

Comment (bganapa) [reply src/tree/StorageAccountTreeItem.ts:132]: in other places as well

**Comment by EricJizbaMSFT** [comment]
Please re-open this or submit a new PR once you're ready to be out of "draft"

**Pull request closed without merging**